packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -128,6 +140,8 @@ let callInternal = (call: functionCall, environment, reducer: ExpressionT.reduce
   | ("keep", [EvArray(aValueArray), EvLambda(aLambdaValue)]) =>
     doKeepArray(aValueArray, aLambdaValue)
   | ("map", [EvArray(aValueArray), EvLambda(aLambdaValue)]) => doMapArray(aValueArray, aLambdaValue)
+  | ("mapSamples", [EvDistribution(SampleSet(dist)), EvLambda(aLambdaValue)]) =>
+    doMapSampleSetDist(dist, aLambdaValue)
```

Comment:
I remember Sam saying otherwise, but this order seems good to me. Data-first, similar to Rescript. When we add the pipe operator, it will act on the first argument.

https://docs.google.com/document/d/1iGMHfsAFmVOJ6Z_4AJugRA-qshuLe-0PkqiRYDULf8Y/edit#heading=h.otq45f3ewbgj